install/kubernetes/istio-rbac-beta.yaml

```diff
@@ -3,6 +3,9 @@
 # Also helps to enable logging on apiserver 'wrap' to see the URLs.
 # Each RBAC deny needs to be mapped into a rule for the role.
 # If using minikube, start with '--extra-config=apiserver.Authorization.Mode=RBAC'
+#
+# NOTE: If deploying istio to a namespace other than 'default' then change the
+# ClusterRoleBinding namspace target appropriately.
```

Comment:
@costinm the Kubernetes default is that service accounts are created in the `default` namespace. For the vast majority of users, they won't need to edit anything. 

If they want to run istio in a different namespace however, then they'll need ensure the namespace of the service account used by `ClusterRoleBinding` matches.

I think part of the confusion is that `ClusterRoleBinding` is scoped to the entire cluster—not a namespace—while it depends on a service account that is scoped to a namespace.